compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -5776,6 +5776,10 @@ private BSymbol findXMLNamespaceFromPackageConst(String localname, String prefix
             return null;
         }
 
+        if (pkgSymbol.getKind() == SymbolKind.PACKAGE && !PackageID.isLangLibPackageID(pkgSymbol.pkgID)) {
```

Comment:
```suggestion
        if (!PackageID.isLangLibPackageID(pkgSymbol.pkgID)) {
```
Do we need to check the symbol kind since we are already passing a package symbol?